zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -130,6 +130,34 @@ pub trait GetBlockTemplateRpc {
         hex_data: HexData,
         _options: Option<submit_block::JsonParameters>,
     ) -> BoxFuture<Result<submit_block::Response>>;
+
+    /// Returns mining-related information.
+    ///
+    /// zcashd reference: [`getmininginfo`](https://zcash.github.io/rpc/getmininginfo.html)
+    #[rpc(name = "getmininginfo")]
+    fn get_mining_info(&self) -> BoxFuture<Result<types::get_mining_info::Response>>;
+
+    /// Returns the estimated network solutions per second based on the last n blocks.
```

Comment:
I think this applies to all 3 RPCs. 

(It probably belongs on the solps field for `get_mining_info::Response`, rather than the whole method.)

Actually we might want to quote the RPC reference instead, what do you think?

```suggestion
    /// Returns the estimated network solutions per second based on the last `num_blocks` before `height`.
    /// If `height` is not supplied, uses the tip height.
    /// If `num_blocks` is not supplied, uses 120 blocks.
```

Reply:
I think this is a bit more descriptive. I applied this suggestion in [`40f180a`](https://github.com/ZcashFoundation/zebra/pull/5808/commits/40f180a319ecc3626f2b9455226d57550a534137).